README.md

```diff
@@ -134,7 +135,7 @@ suggusted VSC extentions:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/GeekyHacks/blog_website_practice/issues).
+Feel free to check the [issues page](https://github.com/GeekyHacks/blog_website/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly fix the broken link to the MIT license under the `License` section so that clicking on the link takes the user to the License agreement file for a more professional readme file.
![image](https://github.com/GeekyHacks/blog_website/assets/81932478/fa835b54-5575-46f9-b0e9-021b314f6a97)